### PR TITLE
add more tests to replication activity

### PR DESCRIPTION
**Pull request by garrensmith**

### CODE REVIEW

File: app/addons/replication/tests/nightwatch/replicationactivity.js [hunk 1]
```diff
@@ -32,18 +32,67 @@ module.exports = {
...
+    client
+      .deleteDatabase('_replicator')
+      .createDatabase('_replicator')
+      .createDocument(replicatorDoc._id, '_replicator', replicatorDoc)
+      .loginToGUI()
+      .waitForElementNotPresent('.global-notification .fonticon-cancel', waitTime, false)
+      .url(baseUrl + '/#replication')
+      .waitForElementNotPresent('.load-lines', waitTime, true)
+      .waitForElementPresent('.replication__filter', waitTime, true)
+      .click('a[title="Edit replication"]')
```
Comment (robertkowalski) [inline app/addons/replication/tests/nightwatch/replicationactivity.js]: change to `clickWhenVisible`

File: app/addons/replication/tests/nightwatch/replicationactivity.js [hunk 2]
```diff
@@ -32,18 +32,67 @@ module.exports = {
       .url(baseUrl + '/#replication')
       .waitForElementNotPresent('.load-lines', waitTime, true)
       .waitForElementPresent('.replication__filter', waitTime, true)
-      .click('a[title="Delete document existing-doc-id-2"]')
-      .waitForElementPresent('.replication_delete-doc-modal', waitTime, true)
-      .click('.replication_delete-doc-modal button.save')
-      .waitForElementNotPresent('.replication_delete-doc-modal', waitTime, true)
-      .waitForElementPresent('.global-notification .fonticon-cancel', waitTime, false)
+      .click('a[href="#/database/_replicator/existing-doc-id-view-doc"]')
```
Comment (robertkowalski) [inline app/addons/replication/tests/nightwatch/replicationactivity.js]: change to `clickWhenVisible`

**Comment by robertkowalski** [comment]
+1 when green

**Comment by garrensmith** [comment]
Tests green. Merged

**Pull request closed without merging**